.travis.yml

```diff
@@ -5,8 +5,7 @@ cache:
 notifications:
   email: false
 node_js:
-  - '7'
-  - '6'
```

Comment:
Please also restore v6; dropping tests is dropping support, and we should be testing on v6 at least until it's not LTS.